fendermint/app/src/cmd/mod.rs

```diff
@@ -105,3 +110,19 @@ fn settings(opts: &Options) -> anyhow::Result<Settings> {
 
     Ok(settings)
 }
+
+pub(crate) fn set_network(network: u8) {
+    let network = Network::from_u8(network).unwrap();
+    set_current_network(network);
+}
+
+pub(crate) fn set_network_from_env() -> anyhow::Result<()> {
+    let network_raw = std::env::var("NETWORK")
+        // default to testnet
```

Comment:
Please let's add an option [exactly like the log_level](https://github.com/consensus-shipyard/fendermint/blob/48a70ce5c10d8519bffe30b01e7fd9e39eaa433c/fendermint/app/src/options/mod.rs#L46-L48), like I asked :pray: 

The problem with this is manifold:
* You changed the default from main to testnet, when `running.md` is already full of `f0...` examples. Why should testnet be the default when the `Network::Mainnet` is the default. It sounds like a choice made because it's convenient for you with IPC testing with Eudico, rather than something that our users will ever want. It's inconsistent with the underlying type. 
* It is a hidden environment variable that nobody will ever know about unless they read the code _and_ find their way into `cmd/mod.rs`.
* All the env vars which are configuration file based start with `FM_`, with a few exceptions: `LOG_LEVEL` maybe a sort of "universal" env var that users might expect and get right in any application; `TENDERMINT_RPC_URL` already existed in the library I think, but it tells you exactly what it is. 
* Compared to those `NETWORK=1` is super cryptic, and the chance that anyone knows to set it is very low. 
* You need this env var in every terminal you open, and remember that this global affects all the address queries, so any CLI interaction where this value doesn't exist will end in frustration.

Whereas if you moved this to be a CLI parameter, then:
* It is discoverable with `--help`, rather than not something hidden in the codebase
* We can give it a suitable env var name such as `LOTUS_NETWORK` or whatever that helps recognise it for what it is.
* Give it a `default_value=0` to maintain backwards compatibility and match `Network`
* We can pass it as `--network=1` or use the env var in our infra scripts with Eudico. We should suffer, not our users.